SimulationAgent/DeviceProperties/DevicePropertiesActor.cs

```diff
@@ -81,16 +94,26 @@ public enum ActorEvents
         /// </summary>
         public IDeviceClient Client => this.deviceConnectionActor.Client;
 
+        /// <summary>
+        /// Failed device twin updates counter
+        /// </summary>
+        public long FailedTwinUpdatesCount {
+            get => this.deviceConnectionActor.FailedTwinUpdatesCount;
+            set => this.deviceConnectionActor.FailedTwinUpdatesCount = value;
```

Comment:
I think it would be easier to keep the value here, and update the simulation runner (see `SimulationErrorsCount`) to pull this value directly from this actor.  So we avoid this actor writing a value into the other actor, which makes the flow hard to follow